### DISCUSSION TITLE
Size Workplane not updating.

### DISCUSSION THREAD

**Discussion by GitBruno** [dune3d/dune3d#132] [General]
When I select a Workplane and change the Width and Height in the Selection panel. The Work plane is not updated. When I deselect the Workplane and select it again I can see values stay at 10mm. How can I fix this?

**Comment by carrotIndustries** [comment]
Which version are you using? In 1.1, you need to press enter to commit the entered value. This has since been fixed on `main`.

**Comment by GitBruno** [comment]
I was using 1.1.0 but did not build from Main. That solved the issue 👍 Thanks!
